src/Link.cc

```diff
@@ -78,6 +80,8 @@ class sdf::Link::Implementation
   /// \brief True if this link should be subject to wind, false otherwise.
   public: bool enableWind = false;
 
+  public: bool autoInertiaSaved = false;
```

Comment:
Document this variable

Reply:
Added in [af8052e](https://github.com/gazebosim/sdformat/pull/1299/commits/af8052ec8faf58206660a1fa508b58b7ccfbefe0)